Source/ACE.Server/Mods/ModManager.cs

```diff
@@ -0,0 +1,293 @@
+using ACE.Server.Managers;
+using ACE.Server.WorldObjects;
+using log4net;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace ACE.Server.Mods
+{
+    public static class ModManager
+    {
+        private static readonly ILog log = LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+
+        /// <summary>
+        /// Mods with at least metadata loaded
+        /// </summary>
+        private static List<ModContainer> Mods { get; set; } = new();
+
+        #region Init / Shutdown
+        public static void Initialize()
+        {
+            FindMods();
+        }
+
+        internal static void Shutdown()
+        {
+            //Todo: consider 
+            DisableAllMods();
+        }
+        #endregion
+
+        #region Load Metadata
+        /// <summary>
+        /// Finds all valid mods in the mod directory and attempts to load them.
+        /// </summary>
+        public static void FindMods()
+        {
+            var modPath = Common.ConfigManager.Config.Server.ModsDirectory ??
+                Path.Combine(Directory.GetParent(Common.ConfigManager.Config.Server.DatFilesDirectory).Parent.FullName, "Mods");
```

Comment:
```suggestion
                Path.Combine(Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location), "Mods");
```

Reply:
Apologies for the delayed feedback

When I tried running this latest version awhile ago, it created a C:\Mods\ folder on my system. My DatFilesDirectory is c:\ac\.

I don't think it's the best form for this to go 1 level up from the Dat folder

I think a better place for code mods would be in a Mods folder that sits next to the ACE.Server.exe. This would be similar to the Content folder that sits next to it

Reply:
Happy to go with whatever you prefer.  I've got everything sitting under C:\ACE\